### ISSUE TITLE
[Suggestion]: alert is showed always on main page

### ISSUE THREAD

**Issue by olekstomek** [miskibin/sejm-stats#48]
### Suggestion / Feature Request

Alert on right down area with message "Ostatnia aktualizacja"
`body > div.flex.overflow-hidden.bg-neutral-50.dark\:bg-neutral-700 > div.flex.flex-col.flex-1.overflow-hidden > main > div > div.fixed.bottom-4.right-4.space-y-2.z-50 > div > div`
is showed on main page (https://sejm-stats.pl/) even if I closed alert dialog manually. I think it is not very important information and it can be put on footer for example. 
Similar alert is about donation but I know that it is important. 
To sum up - the idea is to show only one message about donation.
